src/main/java/mate/academy/springboot/swagger/util/SortUtil.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.springboot.swagger.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SortUtil {
+    private static final int DIRECTION_INDEX = 1;
+    private static final int PARAM_INDEX = 0;
+
+    public static Sort sort(String sortBy) {
```

Comment:
```suggestion
    public Sort sort(String sortBy) {
```
don't use static if it's possible, it's a bad practice